content/en/authors/walentynowicz/_index.md

```diff
@@ -14,50 +15,61 @@ role: PhD Student
 
 # Organizations/Affiliations
 organizations:
-- name: Department of Computational Intelligence, Wroclaw University of Science and Technology
-  url: ""
+  - name: Department of Computational Intelligence, Wroclaw University of Science and Technology
+    url: ""
 
 # Short bio (displayed in user profile at end of posts)
-bio: My research interests include NLP.
+bio: NLP Researcher, Machine Learning Engineer, Python Developer.
 
 interests:
+  - Morphological Tagging
+  - Sequence Tagging
+  - Colloquial and Social Media Texts
+  - Functional Programing
 
 education:
   courses:
-  - course: MsC in Computer Science
-    institution: Wroclaw University of Science and Technology
-    year: 2008
+    - course: PhD in Computer Science, Artificial Intelligence
+      institution: Wroclaw University of Science and Technology
+      year:
```

Comment:
dodaj estimated w przyszłości kiedy (tak inni tutaj dodawali)

Reply:
> dodaj estimated w przyszłości kiedy (tak inni tutaj dodawali)

Wzorowałem się na pliku Łukasza Augustyniaka i on też nie dodawał. Jak dotrę do kompa poprawie.

Reply:
to może nakłamałem, wydawało mi się że wpisywaliśmy